Makefile

```diff
@@ -84,32 +86,27 @@ stop: .env
 	docker-compose stop
 
 .PHONY: test
-test: .env .docker-build
-	@bin/test.sh
+test: .env .docker-build frontend/build/
+	bin/test.sh
 
 .PHONY: testshell
 testshell: .env .docker-build
-	@bin/test.sh --shell
+	bin/test.sh --shell
 
 .PHONY: run
 run: .env .docker-build
 	docker-compose up web worker frontend
 
-.PHONY: gunicorn
-gunicorn: .env .docker-build
-	docker-compose run --service-ports web web
-
-.PHONY: django-shell
-django-shell: .env .docker-build
-	docker-compose run web python manage.py shell
-
 .PHONY: docs
 docs:
-	@bin/build-docs-locally.sh
+	bin/build-docs-locally.sh
+
+frontend/build/:
+	make build-frontend
```

Comment:
This reduces a step in local development. Having said that, when this is run in CI, it causes problems because the check for this directory happens on the host and not in the container. I'm going to leave it here, but it needs some thinking.